src/qs_localization_methods.F

```diff
@@ -365,6 +370,11 @@ SUBROUTINE jacobi_rotations_serial(weights, zij, vectors, max_iter, eps_localiza
       CALL cp_fm_get_info(rmat, nrow_global=nstate)
       tolerance = 1.0e10_dp
 
+      IF (restricted > 0) THEN
+         WRITE (6, *) "JACOBI: The last ", restricted, " orbitals do not rotate"
```

Comment:
Note CP2K's coding convention [#c012](https://www.cp2k.org/conv#c012) concerning hardwired unit numbers.

Reply:
Sorry, I missed it. Replaced it in the code with cp_logger_get_default_unit_nr().